proto/asn1-pdu/asn1_universal_types_to_der.cc

```diff
@@ -39,24 +39,20 @@ void Encode(const Boolean& boolean, std::vector<uint8_t>& der) {
 }
 
 void Encode(const Integer& integer, std::vector<uint8_t>& der) {
-  // Save the current size in |tag_len_pos| to place tag and length
-  // after the value is encoded.
-  const size_t tag_len_pos = der.size();
+  der.push_back(kAsn1Integer);                                // Tag
+  der.push_back(std::min<uint8_t>(1, integer.val().size()));  // Length
```

Comment:
Ah, sorry for the confusion here. This is entirely my fault.

I was meaning you could call
```suggestion
EncodeTagAndLength(kAsn1Integer, der.size(), std::min(1u, integer.val.size()), der);
```

here, since the length may be > 1 byte.

Reply:
No worries! That looks even better.

Reply:
@sleevi it should be good now.